packages/fake-plugin/lib/plugin.js

```diff
@@ -58,22 +98,47 @@ class FakePlugin extends BasePlugin {
     return originalRes;
   }
 
+  /**
+   * Gets the fake session data, adding a delay.
+   *
+   * @param {Function} next - The function to be executed after this one
+   * @param {Object} driver - The driver instance. Using Object because fakesession doesn't exist in BaseDriver
+   * @returns {Promise<Object?>}
+   */
   async getFakeSessionData(next, driver) {
     await B.delay(1);
     return driver.fakeSessionData || null;
   }
 
+  /**
+   *
+   * @param {Function} next - The function to be executed after this one
+   * @param {Object} driver - The driver instance. Using Object because fakesession doesn't exist in BaseDriver
+   * @param {any[]} args - Additional arguments to be passed to the next function
+   * @returns {Promise<null>}
+   */
   async setFakeSessionData(next, driver, ...args) {
     await B.delay(1);
     driver.fakeSessionData = args[0];
     return null;
   }
 
+  /**
+   *
+   * @param {Function} next - The function to be executed after this one
+   * @returns {Promise<string>}
+   */
```

Comment:
This should fail the check for a `PluginCommand` because `driver` isn't present.  Given it's not absolutely necessary, maybe we should make that parameter optional in the `PluginCommand` type.